org-code-javabuilder/lib/src/main/java/org/code/javabuilder/JavaBuilder.java

```diff
@@ -0,0 +1,66 @@
+package org.code.javabuilder;
+
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.services.apigatewaymanagementapi.AmazonApiGatewayManagementApi;
+import com.amazonaws.services.apigatewaymanagementapi.AmazonApiGatewayManagementApiClientBuilder;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder;
+
+import java.io.IOException;
+
+public class JavaBuilder {
+  private final OutputHandler outputHandler;
+  private final InputPoller inputPoller;
+  private final OutputPoller outputPoller;
+  private final JavaRunner javaRunner;
+  private final OutputSemaphore outputSemaphore;
+
+  public JavaBuilder(String connectionId, String apiEndpoint, String queueUrl) {
```

Comment:
Is there a better way to structure this? Here, we're just instantiating a bunch of local variables to inject them into the other classes. Is that the right way to do it?